web/src/user_group_edit.js

```diff
@@ -285,6 +285,18 @@ function update_group_membership_button(group_id) {
     }
 }
 
+export function handle_subgroup_edit_event(group_id) {
+    if (!overlays.groups_open()) {
+        return;
+    }
+    const group = user_groups.get_user_group_from_id(group_id);
+
+    // update members list if currently rendered.
+    if (is_editing_group(group_id)) {
+        user_group_edit_members.update_member_list_widget(group);
+    }
```

Comment:
There is still some work needed here for live-update like permissions to manage a group can also change if a subgroup is added to a group.

Reply:
Do you mean specifically that whether you have permission to edit the other settings may need to live-update? Yeah, that makes sense. Hopefully that code does not end up super fiddly.

Reply:
Yes. Adding or removing a subgroup may result in changing permissions if the modified group is used for a setting.

Reply:
Hmm, yeah. That may prove to be messy to trace through all the things that might need to rerender. 
Let's open an issue to track that, as it may be a longer project that needs some strategy.

Reply:
Opened #32089 to track this.